book-worm-2010/book-worm-fe
# Feature/header search style

### What does this PR do? 
>Clearly and concisely describe the feature. 
This PR adds error handling for the search book form as well as styles the modal. It also adds a home button to header but it is not linked yet.

### Is this a feature or Fix/Refactor?  
Feature

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/Header/Header.js` lines: 17-20;
`src/NewBookForm/NewBookForm.js` lines: 5, 10-13, 24-29, 31-55;
`src/NewBookForm/_NewBookForm.scss' lines: 25-35, 40-53;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` the branch
2. run `npm start`
3. Click on add a book mountain button
4. Click submit and the error message will appear
